src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -662,6 +662,9 @@ private void ProcessNode(ref BioProcesses processor, ref CompoundStorage storage
         if (environmentModifier <= MathUtils.EPSILON)
             canDoProcess = false;
 
+        if (process.SpeedMultiplier <= 0)
```

Comment:
Laws of physics will break if the speed multiplier is over 1 so I'd add an else here that makes sure speed multiplier can never be over 1.

Reply:
Would regular cap do (maximize at 1)?

Reply:
What do you mean? My suggestion is have this code:
```c#
if (process.SpeedMultiplier <= 0)
{
    canDoProcess = false;
}
else if (process.SpeedMultiplier > 1)
{
    process.SpeedMultiplier = 1;
}
```